annotator/static/views/player.js

```diff
@@ -46,6 +46,12 @@ class PlayerView {
         // Video URL
         this.videoSrc = videoSrc;
 
+        // Video start time
+        this.videoStart = parseFloat(videoStart);
```

Comment:
I ran into a bug from it. videoStart and videoEnd was a string. I figured it was from using a query string. Unless is there some configuration in the backend I should've done?